vega_sim/reinforcement/market_sim.py

```diff
@@ -5,187 +5,71 @@
 import torch
 import time
 
-
+from vega_sim.environment.agent import Agent
 from vega_sim.reinforcement.learning_agent import (
     Action,
     LearningAgent,
-    MarketState,
+    LAMarketState,
     WALLET as LEARNING_WALLET,
 )
-from vega_sim.environment.agent import Agent
+
+from vega_sim.scenario.registry import IdealMarketMakerV2
+from vega_sim.parameter_test.parameter.loggers import (
+    ideal_market_maker_single_data_extraction,
+    target_stake_additional_data,
+    v1_ideal_mm_additional_data,
+    tau_scaling_additional_data,
```

Comment:
Suspect most of these imports are un-needed?

Reply:
Yes, I can clean this up.